### ISSUE TITLE
Require Cycle warning

### ISSUE THREAD

**Issue by aminmeshk** [MohamadKh75/react-native-jalali-datepicker#5]
Every time the app renders the date picker, it gives the warning below (in the console) in React Native v0.63 and I think there should be an easy fix for it:

```
Require cycle: node_modules\@mohamadkh75\react-native-jalali-datepicker\src\components\index.js -> node_modules\@mohamadkh75\react-native-jalali-datepicker\src\components\YearSelector.js -> node_modules\@mohamadkh75\react-native-jalali-datepicker\src\components\index.js

Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle.
```

**Comment by aminmeshk** [comment]
Oh and also it gives the same warning but for utils.js, Calendar.js and MonthSelector.js.
(For example: index.js > Calendar.js > index.js)
They all are using index.js but they shouldn't.

**Comment by aminmeshk** [comment]
I fixed it and created a pull request: #6 :)
